helpers/postgres.rb

```diff
@@ -121,7 +121,7 @@ def generate_postgres_options(flavor: nil, location: nil)
       end
     end
 
-    options.add_option(name: "version", values: Option::POSTGRES_VERSION_OPTIONS)
+    options.add_option(name: "version", values: Option::POSTGRES_VERSION_OPTIONS[flavor || PostgresResource::Flavor::STANDARD])
```

Comment:
I think this should be:

```ruby
    options.add_option(name: "version", values: Option::POSTGRES_VERSION_OPTIONS.values.flatten.uniq, parent: "flavor") do |flavor, version|
      Option::POSTGRES_VERSION_OPTIONS[flavor].include?(version)
    end
```

Two important changes:
- We add flavor as parent, otherwise it wouldn't create the tree with correct hierarchy
- Instead of filtering at the `values`, we do filtering at the block. Otherwise you would show PG18 for lantern and paradedb as well (at the creation form, not at the update forms).

Reply:
FYI, after doing this, you would need to update version in ContentGenerator like this;

```ruby
    def self.version(flavor, version)
      "Postgres #{version}"
    end
```